cmd/convox/Makefile

```diff
@@ -1,51 +1,28 @@
-.PHONY: all build clean package package-deps package-export package-gopath release
+.PHONY: all build package release
 
-pkg_darwin  = pkg/convox-darwin-10.12-amd64
-pkg_darwin_arm64  = pkg/convox-darwin-10.12-arm64
+pkg_darwin  = pkg/convox-darwin-amd64
+pkg_darwin_arm64  = pkg/convox-darwin-arm64
 pkg_linux   = pkg/convox-linux-amd64
-pkg_linux_arm64   = pkg/convox-linux-amd64
-pkg_windows = pkg/convox-windows-4.0-amd64.exe
-
-sources = $(shell find . -name '*.go')
+pkg_linux_arm64   = pkg/convox-linux-arm64
+pkg_windows = pkg/convox-windows-amd64.exe
 
 all: build
 
 build:
 	go install ./...
 
-clean:
-	rm -f pkg/convox-*
-
-package: package-deps package-gopath $(pkg_darwin) $(pkg_linux) $(pkg_windows) package-export
-
-package-deps:
-	go install github.com/crazy-max/xgo@v0.7.5
-	mkdir -p pkg
-
-# copy finished packages out if running inside a container
-package-export:
-	if [ -f /.dockerenv ]; then mkdir -p pkg && docker run -v $(GOPATH):/gopath -i ubuntu tar czv /gopath/src/github.com/convox/rack/cmd/convox/pkg | tar xzv -C pkg --strip-components 8; fi
-
-# copy gopath in if running inside a container
-package-gopath:
-	if [ -f /.dockerenv ]; then tar cz $(GOPATH) | docker run -v $(GOPATH):/gopath -i ubuntu tar xz -C /gopath --strip-components 2; fi
+package:
+	mkdir -p pkg/
+	export CGO_ENABLED=0
+	env GOOS=linux GOARCH=amd64 go build -ldflags "-X main.version=$(VERSION)" -o $(pkg_linux)
+	env GOOS=linux GOARCH=arm64 go build -ldflags "-X main.version=$(VERSION)" -o $(pkg_linux_arm64)
+	env GOOS=darwin GOARCH=amd64 go build -ldflags "-X main.version=$(VERSION)" -o $(pkg_darwin)
+	env GOOS=darwin GOARCH=arm64 go build -ldflags "-X main.version=$(VERSION)" -o $(pkg_darwin_arm64)
+	env GOOS=windows GOARCH=amd64 go build -ldflags "-X main.version=$(VERSION)" -o $(pkg_windows)
```

Comment:
The `xgo` was building a damaged executable on _darwin/arm64_, changed to use the native build system.